.github/workflows/release-tagger.yml

```
@@ -0,0 +1,37 @@
+      - name: Check PR title with regex
+        id: check_pr_title
+        run: |
+          if [[ "${{ github.event.pull_request.title }}" =~ $REGEX_PATTERN ]]; then
+            echo "match=true" >> $GITHUB_ENV
+          else
+            echo "match=false" >> $GITHUB_ENV
+          fi
+      - name: Create tag
+        if: env.match == 'true'
+        id: create_tag
+        run: |
```

actionlint (1.7.4)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)